pybricksdev/cli/__init__.py

```diff
@@ -127,7 +127,8 @@ def add_parser(self, subparsers: argparse._SubParsersAction):
         parser.add_argument(
             "conntype",
             metavar="<connection type>",
-            help="connection type: %(choices)s",
+            help="connection type: %(choices)s; "
```

Comment:
I think I would rather keep the help short and document the details at <https://docs.pybricks.com/projects/pybricksdev> (i.e. add a new section explaining the `pybricksdev` command line too and all of the commands). Otherwise we should also have to add a note here about `ssh` only working with EV3 running ev3dev and `ble` not working on EV3 to be complete.

Reply:
I think I'll leave this one for you. You know the best what the tool is compatible with rather than me.